.gitignore

```diff
@@ -1,2 +1,3 @@
 dist/
-*.wasm
\ No newline at end of file
+*.wasm
+tools/analyzers/analyzers
```

Comment:
Why is this added?

Reply:
I wasn't able to figure out how to run the analyzers with `go run` with the right directory context, so instead it builds them and then runs them. This ignores the built binary

Reply:
I was able to get the analyzers to run directly without pre-compiling, so I removed this line.

The only things left that aren't directly `go run` from the module are the yaml and markdown linter, which are docker containers.